Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -52,6 +57,21 @@ + (void)application:(UIApplication *)application didRegisterUserNotificationSett
   }
 }
 
++ (void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken
```

Comment:
When I trying to use this fork, this line seems to be never called. Did I miss anything?